src/ui/server/controllers/insurance/eligibility/eligible-to-apply-online/index.test.ts

```diff
@@ -105,31 +94,32 @@ describe('controllers/insurance/eligibility/eligible-to-apply-online', () => {
         expect(res.redirect).toHaveBeenCalledWith(expected);
       });
 
-      describe('when there is no application', () => {
+      describe('when the application.create call fails', () => {
         beforeEach(() => {
-          // @ts-ignore
-          createApplicationSpy = jest.fn(() => Promise.resolve());
+          req.session = {
+            ...req.session,
+            submittedData: {
+              ...req.session.submittedData,
+              insuranceEligibility: mockSession.submittedData.insuranceEligibility,
+            },
+          };
 
-          api.keystone.application.create = createApplicationSpy;
+          createApplicationSpy = jest.fn(() => Promise.reject(new Error('mock')));
```

Comment:
could create mockErrorMessage like in API and then important use that here

Reply:
Agree - can we do this in a different PR though, there are 195 instances of this.

Reply:
Created EMS-3600 to do this